modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/representative_users_controller.rb

```diff
@@ -4,7 +4,35 @@ module AccreditedRepresentativePortal
   module V0
     class RepresentativeUsersController < ApplicationController
       def show
-        render json: @current_user
+        # TODO: Once we figure out how we're handling serialization and which library we're using,
+        # moving this serialization logic out to to a serialization layer.
+        render json: {
+          account: {
+            account_uuid: @current_user.uuid
```

Comment:
Do you need a uuid to be returned here? It's not a huge deal if you specifically want this uuid for some reason, but generally the `@current_user.user_account.id` is preferred

Reply:
Yes they have a `user_account`, it's how the ICN gets reloaded on the RepresentativeUser redis object

Reply:
This is one way to find it: `UserAccount.find_by(icn: representative_user.icn)`, but it should probably be added directly in the `representative_user_loader` and `representative_user` classes:

In `representative_user_loader.rb`, where the attributes are being copied to the `RepresentativeUser` Redis class:
`user.user_account_uuid = session.user_account.id`

And then in `representative_user.rb`:
`attribute :user_account_uuid`

Then you'll be able to use it off the `RepresentativeUser` model

Reply:
@bosawt I see that we're doing some error logging in [`SubmitController`](https://github.com/department-of-veterans-affairs/vets-website/blob/4477fb0d0c5e1f3ea7cfd75cb6e49b0e01ae7204/src/platform/forms-system/src/js/review/SubmitController.jsx#L96-L147) that uses this `accountUuid` as metadata. Thoughts?

Reply:
Ah I see. The `accountUuid` on a normal `User` is actually `user.account`, which is a deprecated field, it prob shouldn't be used on the FE at all (although I see it's used in various places for metadata and to send to Sentry, etc). A `RepresentativeUser` definitely doesn't have a `user.account`, so `user.account.id` is not a possibility. However, I still think it'd be best to user `user.user_account.id` (The `UserAccount`, separate from the deprecated `Account` object). The reason is if you're trying to correlate Sentry or Datadog logs to this action, it is much easier to retrieve information with `UserAccount.find(<user_account_uuid>)`, rather than the `user.uuid`, which could be either `idme_uuid`, or `logingov_uuid`, without an easy way to know which is which (and you won't be able to do `RepresentativeUser.find(<user_uuid>)`, because the `RepresentativeUser` model is ephemeral and will be gone if the user logs out)